### ISSUE TITLE
`display: flex;` the key in `/other/periodic`

### ISSUE THREAD

**Issue by trinkey** [trinkey/website#18]

**Comment by trinkey** [comment]
should probably class it
```html
<section class="key">
  <div style="display: flex; flex-wrap: wrap; column-gap: calc(15 * var(--base-size))">
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal nonmetal"></div> Nonmetals</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal noble"></div> Noble Gases</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal alkali"></div> Alkali Metals</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal alkaline"></div> Alkaline Earth Metals</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal metalloid"></div> Metalloids</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal posttrans"></div> Post-transitional Metals</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal trans"></div> Transition Metals</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal lanth"></div> Lanthanides</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal actinide"></div> Actinides</div>
    <div style="white-space: nowrap;"><div style="display: inline-block;" class="keyVal unknown"></div> Unknown Properties</div>
  </div>
</section>
```

**Comment by trinkey** [comment]
fixed with v2.2.2
